### PR TITLE
Remove bundleconfig.json files from project templates

### PR DESCRIPTION
**Pull request by jbagga**
Addresses #326

### CODE REVIEW

**Comment by jbagga** [comment]
@DamianEdwards should the comment `Please see documentation at https://docs.microsoft.com/aspnet/core/client-side/bundling-and-minification for details on configuring this project to bundle and minify static web assets.` be added to the site.min.css and site.min.js files as well? I can undo that if not.

File: src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/wwwroot/css/site.css [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/*Please see documentation at https://docs.microsoft.com/aspnet/core/client-side/bundling-and-minification for details on configuring this project to bundle and minify static web assets.*/
```
Comment (Eilon) [inline src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/wwwroot/css/site.css]: Recommend making this a shorter line (i.e. split into 2-3 shorter lines).

**Comment by Eilon** [comment]
I think once the file is minified the comment would be gone anyway, so I wouldn't add it there.

Though, perhaps more interestingly, it's a bit odd to have a site.min.css because nothing will keep it up-to-date...

File: src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/wwwroot/js/site.js [hunk 2]
```diff
@@ -1 +1,2 @@
-﻿// Write your Javascript code.
+﻿//Please see documentation at https://docs.microsoft.com/aspnet/core/client-side/bundling-and-minification for details on configuring this project to bundle and minify static web assets.
```
Comment (Eilon) [inline src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/wwwroot/js/site.js]: Space after the `//`

**Comment by jbagga** [comment]
> nothing will keep it up-to-date...

If there are ever changes made to the `site.js` or `site.css`, shouldn't the developer be responsible for creating/updating the min files?

**Comment by Eilon** [comment]
@jbagga yes exactly, except we don't do it by default, so apps don't work by default. But @DamianEdwards wants to keep the file there to keep the pattern going. So yeah let's keep it. But I don't think we need the comment in the min files.

File: src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/wwwroot/js/site.js [hunk 3]
```diff
@@ -1 +1,3 @@
-﻿// Write your JavaScript code.
+﻿// Please see documentation at https://docs.microsoft.com/aspnet/core/client-side/bundling-and-minification
+// for details on configuring this project to bundle and minify static web assets.
+// Write your JavaScript code.
```
Comment (Eilon) [inline src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/wwwroot/js/site.js]: Maybe put 1 blank line between the new comment and the old comment?

File: src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-FSharp/wwwroot/js/site.js [hunk 4]
```diff
@@ -1 +1,3 @@
-﻿// Write your Javascript code.
+﻿// Please see documentation at https://docs.microsoft.com/aspnet/core/client-side/bundling-and-minification
+// for details on configuring this project to bundle and minify static web assets.
+// Write your Javascript code.
```
Comment (Eilon) [inline src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-FSharp/wwwroot/js/site.js]: See above comment.

**Comment by Eilon** [comment]
:shipit:

**Pull request merged**